src/Web/wwwroot/app/accounts/views/accountsLoginInfo.html

```diff
@@ -8,7 +8,7 @@
     </div>
     <div class="form-group has-feedback" show-errors>
         <label for="email" class="sr-only">Email</label>
-        <input type="email" id="email" name="Email" class="form-control" placeholder="Email" ng-model="model.email"
+        <input type="email" id="email" name="Email" class="form-control" placeholder="Email" ng-model="model.email" autofocus
```

Comment:
Does this work? I can't fond anything about an `autofocus` directive being part of angularjs without writing something custom...